consensus/src/quorum_store/quorum_store_builder.rs

```diff
@@ -392,6 +392,7 @@ impl InnerBuilder {
                     match aptos_db_clone.get_latest_ledger_info() {
                         Ok(ledger_info) => BatchResponse::NotFound(ledger_info),
                         Err(e) => {
+                            let e = anyhow::Error::from(e);
```

Comment:
why not derive `Debug` for our errors?

Reply:
Not sure how debug helps here.

Reply:
then `error = ?e` can render? otherwise why are you defining `e` here?